delatore/sources/awx_api.py

```diff
@@ -42,12 +42,13 @@ class AWXApiSource(Source):
     _params: Optional[AWXParams] = None
 
     async def get_update(self):
-        template_name = await self.client.get(self.polling_interval)
-        if template_name is None:
+        msg = await self.client.get(self.polling_interval)
+        if msg is None:
             raise NoUpdates
 
+        template_name, depth = msg.split(';')
```

Comment:
Split can raise `ValueError` in case there are more `;` than 1, I suppose, this should be handled here and added as negative test